app/views/tmdb/tv_season.html.erb

```diff
@@ -1,17 +1,16 @@
-
-<h1><%= link_to "#{@series.show_name}", tv_series_path(show_id: @show_id) %> <%= @season_number_display %></h1>
-<p class="button-main"><%= link_to "Back to #{@series.show_name}", tv_series_path(show_id: @show_id) %></p>
+<h1><%= link_to "#{@series.show_name}", tv_series_path(show_id: @series.show_id) %> <%= formatted_season_number(@season_number) %></h1>
```

Comment:
The theme of the changes on this page is replacing instance variables with:
* accessing the values from the parent object
* view helper methods